### PR TITLE
Add option to add flow type-checker, with annotated templates

### PR DESCRIPTION
**Pull request by HugoCapocci**

### CODE REVIEW

File: test/flow/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const _ = require('lodash');
```
Comment (ghamaide) [inline test/flow/index.js]: not sure you need lodash here

File: generators/server/index.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ module.exports = generators.Base.extend({
         'cors': '2.8.4',
         'csurf': '1.9.0',
         'debug': '3.1.0',
-        'healthcheck-fastit': 'git+ssh://git@github.com:fastit/health-check.git#1.0.1',
+        'healthcheck-fastit': 'git+ssh://git@github.com/fastit/health-check.git#1.0.1',
```
Comment (ghamaide) [inline generators/server/index.js]: why do you change this ? it works fine with `:`

File: test/flow/index.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+const assert = require('yeoman-assert');
```
Comment (ghamaide) [inline test/flow/index.js]: 👍

File: generators/server/index.js (hunk 2 above)
Comment (HugoCapocci) [reply generators/server/index.js]: with yarn 0.19.1 I had an non-blocking error during my tests -> https://stackoverflow.com/questions/40016023/yarn-unable-to-clone-private-github-repo-ssh-could-not-resolve-hostname-githu/40630175#40630175

**Pull request closed without merging**